source/MaterialXTest/Runtime.cpp

```diff
@@ -463,6 +463,15 @@ TEST_CASE("Runtime: Prims", "[runtime]")
     REQUIRE(!backdrop.getContains().hasTargets());
     backdrop.getNote().getValue().asString() = "These aren't the Droids you're looking for";
     REQUIRE(backdrop.getNote().getValue().asString() == "These aren't the Droids you're looking for");
+    REQUIRE(backdropPrim.getRelationship(backdrop.getContains().getName()) == backdrop.getContains());
```

Comment:
Testing getRelationship and getRelationships APIs.

Reply:
I assume that the non-trivial iterator case is used to generically itererate through the look class which have > 1 relationship ?